src/bin/reset_test_instance.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+echo ""
+echo "Check if commit message requires reset"
+git clone --depth=1 https://github.com/greenpeace/"${CONTAINER_PREFIX}"
+GIT_COMMIT_MSG=$(git --git-dir="${CONTAINER_PREFIX}"/.git log --format=%B -n 1 "$CIRCLE_SHA1")
+if [[ $GIT_COMMIT_MSG == *"[RESET]"* && $CONTAINER_PREFIX == *"test-"* ]]; then
```

Comment:
`make reset` only existins in instances that don't have any environment other that development. But since we have some specific scope test instances outside of our regular swarm, I added one more check here to make sure we don't accidentally reset them (since those don't have the `test-` prefix).